### DISCUSSION TITLE
Removing a note/notes from the index (after they have been indexed)

### DISCUSSION THREAD

**Discussion by FamedBear16** [brianpetro/obsidian-smart-connections#377] [Q&A]
Hi,
is there any way to remove notes from the index. 
I have now excluded my refereces folder form indexing, but all my notes  in this foder are still indexed.

I have also noticed that if I move a note that was index to another folder, the smart connection of another note still points to the old file path

Regards,

**Comment by brianpetro** [comment]
@FamedBear16, thanks for the insight.

V2 implements some new methods for pruning, and there's a button in the settings that will remove these connections without having to "force refresh."

I hope that helps!
🌴
